impeller/renderer/backend/vulkan/test/mock_vulkan.cc

```diff
@@ -153,6 +157,7 @@ void vkGetPhysicalDeviceProperties(VkPhysicalDevice physicalDevice,
       static_cast<VkSampleCountFlags>(VK_SAMPLE_COUNT_1_BIT |
                                       VK_SAMPLE_COUNT_4_BIT);
   pProperties->limits.maxImageDimension2D = 4096;
+  pProperties->limits.timestampPeriod = 1;
```

Comment:
(Just a comment)

We could make this configurable if you want to ensure timestampPeriod of 0 doesn't crash.